go/tasks/pluginmachinery/flytek8s/config/config.go

```diff
@@ -43,6 +43,10 @@ type K8sPluginConfig struct {
 	DefaultCPURequest string `json:"default-cpus" pflag:",Defines a default value for cpu for containers if not specified."`
 	// default memory requests for a container
 	DefaultMemoryRequest string `json:"default-memory" pflag:",Defines a default value for memory for containers if not specified."`
+	// Tolerations for interruptible pods
```

Comment:
can we explain what interruptible means?

Reply:
sure